test/model/transaction/MosaicDefinitionTransaction.spec.ts

```diff
@@ -38,7 +38,7 @@ describe('MosaicDefinitionTransaction', () => {
             Deadline.create(epochAdjustment),
             MosaicNonce.createFromUint8Array(new Uint8Array([0xe6, 0xde, 0x84, 0xb8])), // nonce
             new MosaicId(UInt64.fromUint(1).toDTO()), // ID
-            MosaicFlags.create(true, true, true),
+            MosaicFlags.create(true, true, true, true),
```

Comment:
why do these tests need to change at all? isn't new param optional, in which case they should all pass as before?

Reply:
MosaicFlags Unit tests has been refactored